python/out.gv

```diff
@@ -0,0 +1,21 @@
+digraph erdos_dataflow {
```

Comment:
Was this file included by mistake or are you planning on using it in an example?

Reply:
Added by mistake. Removed it in a follow-up commit.